addons/binding/org.openhab.binding.chromecast/META-INF/MANIFEST.MF

```diff
@@ -7,7 +7,7 @@ Bundle-Version: 2.1.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ClassPath: .,
  lib/protobuf-java-2.6.0.jar,
- lib/api-v2-0.9.4-SNAPSHOT.jar
+ lib/api-v2-0.10.1-SNAPSHOT.jar
```

Comment:
changes were mostly cosmetic and didn't affect functionality.  Regardless I have rebuilt the jar locally from the latest master branch, I have rebased and pushed